src/components/CameraButton.tsx

```diff
@@ -34,19 +33,14 @@ function CameraButton(props: CameraButtonProps) {
         onClick={() => navigateToCamera({preservedState} || {}, goBack)}
         className={classes.buttonContainer}
         variant="contained"
-        color="primary"
-        disableElevation={true}
+        disableElevation
+        fullWidth
       >
         {photoUri ? (
           <img style={{ width: '100%' }} src={photoUri} />
         ) : (
           <div>
-            <Typography color="primary">
-              <PhotoLibraryIcon />
-            </Typography>
-            <Typography variant="h2" color="primary">
-              Add Photo
-            </Typography>
+            <Typography variant="button" color="primary">+ Add Photo</Typography>
```

Comment:
There's now an error state for photos. Don't know if you wanted to add this into the PR as well or not

![image](https://user-images.githubusercontent.com/35501399/114062033-e82a7d80-984b-11eb-8158-5a6113914120.png)

Reply:
Updated to add this!